scheduler/roles/k8s_prepare_services/vars/main.yml

```diff
@@ -14,3 +14,5 @@
 ---
 
 k8s_version: "{{ hostvars['localhost']['k8s_version'] }}"
+containerd_cfg_file_path: "/etc/containerd/config.toml"
```

Comment:
these variables are used in k8s_habana_container_runtime role. so please move the variables to k8s_habana_container_runtime/vars/main.yml

Reply:
Done